src/elm/Main.elm

```diff
@@ -1319,9 +1068,9 @@ refreshPage model =
         Pages.Build org repo buildNumber focusFragment ->
             Cmd.batch
                 [ getBuilds model org repo Nothing Nothing Nothing
-                , refreshBuild model org repo buildNumber
-                , refreshBuildSteps model org repo buildNumber focusFragment
-                , refreshLogs model org repo buildNumber model.steps Nothing
+                , Cmd.map BuildUpdate <| refreshBuild model org repo buildNumber
```

Comment:
we can still control page refreshes from Main.elm (or move it out someday) while using Cmd.map to map the responses back to the Build component